components/org.wso2.carbon.identity.idle.account.identification/src/main/java/org/wso2/carbon/identity/idle/account/identification/models/InactiveUserModel.java

```diff
@@ -0,0 +1,65 @@
+package org.wso2.carbon.identity.idle.account.identification.models;
```

Comment:
License. Check all the files.

Reply:
Addressed with commit c008575c8f41210bfec80da3de58500170ad968d